File: Operadores Relacionais/Rel.py
a = 7
b = 5

print(f"a = {a}, b = {b}\n")

comparisons = [
    ("a < b", a < b),
    ("a > b", a > b),
    ("a <= b", a <= b),
    ("a >= b", a >= b),
    ("a == b", a == b),
    ("a != b", a != b),
]

for expr, value in comparisons:
    print(f"{expr:8} -> {value}")

if a > b:
    print(f"\ndecisao: a ({a}) é maior que b ({b}).")
elif a < b:
    print(f"\ndecisao: a ({a}) é menor que b ({b}).")
else:
    print(f"\ndecisao: a ({a}) é igual a b ({b}).")

File: Operadores Relacionais/Rel.c
#include <stdio.h>

int main() {
    int a = 7;
    int b = 5;

    printf("a = %d, b = %d\n\n", a, b);
    printf("a < b  -> %d\n", a < b);
    printf("a > b  -> %d\n", a > b);
    printf("a <= b -> %d\n", a <= b);
    printf("a >= b -> %d\n", a >= b);
    printf("a == b -> %d\n", a == b);
    printf("a != b -> %d\n", a != b);

    if (a > b) {
        printf("\ndecisao: a (%d) eh maior que b (%d).\n", a, b);
    } else if (a < b) {
        printf("\ndecisao: a (%d) eh menor que b (%d).\n", a, b);
    } else {
        printf("\ndecisao: a (%d) eh igual a b (%d).\n", a, b);
    }
    return 0;
}